rename_images.py

```diff
@@ -0,0 +1,19 @@
+from os import listdir, rename
```

Comment:
The script should also generate the `.txt` files accompanying the images.
I suggest you use `split('_')` to get the components from the name (if there are two components, it's an image in the format WEBCOMPAT-ID_BROWSER.png and you can skip it; if there are more than two components, it's an image in the format WEBCOMPAT-ID_ELEMENT-ID_SEQUENCE-NUM_BROWSER.png and you have to rename it and put in the `.txt` file in order of SEQUENCE-NUM). You can use this https://github.com/marco-c/autowebcompat/blob/master/data_inconsistencies.py#L12 as inspiration

Reply:
Done!